### DISCUSSION TITLE
Problems with the vulkan and vulkan-hpp port

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/vcpkg#22347] [Q&A]
Hey, I am using Visual Studio 2022 and CMake + VCPKG for my project
here is my manifest : 
```
{
  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg/master/scripts/vcpkg.schema.json",
  "name": "cfrowser",
  "version-string": "0.1",
  "dependencies": [
    "imgui",
    "vulkan",
    "vulkan-hpp"
  ]
}
```

Whenever I am configuring cmake within Visual Studio, everything works fine. Except one thing, the <vulkan/vulkan.hpp> has an error  : "Cannot open source file "vulkan/vulkan.h".
Surely, this means that the vulkan port is having issues finding the VULKAN_SDK, or something similar ? 
I do have the SDK installed as well as I do have an environment variable pointing to the path it is installed at.
I'm not sure what's going on, I'd like some help

**Comment by sidy3d** [comment]
I think that you manually have to add the include path to the VS project.
[https://github.com/microsoft/vcpkg/blob/master/ports/vulkan/usage](https://github.com/microsoft/vcpkg/blob/master/ports/vulkan/usage)
